src/lib.rs

```diff
@@ -136,7 +136,13 @@
 )]
 
 #[macro_use]
-extern crate html5ever;
+pub extern crate html5ever;
+
+pub use cssparser;
+pub use ego_tree;
+pub use selectors;
+pub use smallvec;
```

Comment:
Re-exporting crates should be limited to those used in the public API which does not seem to be the case for at least `smallvec`.

Reply:
Agreed, removed the `smallvec` re-export.

Reply:
Feel free to squash the commits =).

Reply:
I started checking the others and for example, I could not find a place where `tendril` is part of our API. Similarly, I don't think we expose any types from `cssparser`. Could you please check that the list really only includes crates exposed in the API. Otherwise, this is an unnecessary commitment w.r.t. backwards compatibility.

Reply:
I can do this in the morning, mainly interested in the cssparser re-export, since versions of it cause breaking changes in the ‘selectors’ crate.

Reply:
Going to close the PR, I think it is fine to keep as is, until it becomes more adherent to extend.